dev/package.json

```diff
@@ -47,22 +47,33 @@
         "eslint": "^7.19.0",
         "eslint-plugin-import": "^2.22.1",
         "glob": "^7.1.6",
-        "mocha": "^9.1.3",
-        "mocha-junit-reporter": "^2.0.2",
-        "mocha-multi-reporters": "^1.1.7",
         "ts-node": "^10.9.1",
-        "typescript": "^4.9.4"
+        "typescript": "^4.9.4",
+        "https-browserify": "^1.0.0",
+        "mocha": "^7.1.1",
+        "mocha-junit-reporter": "^1.23.1",
+        "mocha-multi-reporters": "^1.1.7",
+        "net-browserify": "^0.2.4",
+        "os-browserify": "^0.3.0",
+        "path-browserify": "^1.0.1",
+        "process": "^0.11.10",
+        "querystring-es3": "^0.2.1",
+        "stream-browserify": "^3.0.0",
+        "stream-http": "^3.2.0"
     },
     "dependencies": {
         "@azure/arm-subscriptions": "^2.0.0",
         "@azure/ms-rest-azure-env": "^2.0.0",
         "@azure/ms-rest-js": "^2.2.1",
         "@azure/ms-rest-nodeauth": "^3.0.5",
         "clean-webpack-plugin": "^3.0.0",
-        "copy-webpack-plugin": "^6.0.0",
+        "copy-webpack-plugin": "6.0.0",
         "fs-extra": "^8.0.0",
         "terser-webpack-plugin": "^5.0.0",
         "ts-loader": "^9.4.2",
-        "webpack": "5.28.0"
+        "webpack": "^5.74.0",
+        "console-browserify": "^1.2.0",
```

Comment:
Where are we using console?

Reply:
All those browserify packages were required to get Resource Groups to work right now. I'm debating if I should put all the fallbacks in the dev extension or just let the extensions that need it to import it.

Probably something in between where the most common ones are in dev, and very specific ones are in client extensions.